examples/inverse/dics_epochs.py

```diff
@@ -0,0 +1,86 @@
+
+In this example, a Dynamic Imaging of Coherent Sources (DICS)
+:footcite:`GrossEtAl2001` beamformer is used to transform sensor-level
+time-frequency objects to the source level. We will look at the event-related
+synchronization (ERS) of beta band activity in the :ref:`somato dataset
+<somato-dataset>`.
+"""
+# Authors: Marijn van Vliet <w.m.vanvliet@gmail.com>
+#          Alex Rockhill <aprockhill@mailbox.org>
+# License: BSD-3-Clause
+
+import numpy as np
+import mne
+from mne.datasets import somato
+from mne.time_frequency import tfr_morlet, csd_tfr
+from mne.beamformer import make_dics, apply_dics_tfr_epochs
+
+print(__doc__)
+
+# %%
+# Set up the filenames.
+data_path = somato.data_path()
+subject = '01'
+task = 'somato'
+raw_fname = (data_path / f'sub-{subject}' / 'meg' /
+             f'sub-{subject}_task-{task}_meg.fif')
+fname_fwd = (data_path / 'derivatives' / f'sub-{subject}' /
+             f'sub-{subject}_task-{task}-fwd.fif')
+subjects_dir = data_path / 'derivatives' / 'freesurfer' / 'subjects'
+
+# %%
+# First, we load the data and compute for each epoch the time-frequency
+# decomposition in sensor space.
+
+# Load raw data and make epochs. For speed, we only use the first 15 events.
+raw = mne.io.read_raw_fif(raw_fname)
+events = mne.find_events(raw)[:15]
+epochs = mne.Epochs(raw, events, event_id=1, tmin=-1.5, tmax=2,
+                    preload=True)
+
+# We are mostly interested in the beta band. Define a range of frequencies,
+# using a log scale, from 12 to 30 Hz.
+freqs = np.logspace(np.log10(12), np.log10(30), 9)
+
+# Use Morlet wavelets to compute sensor-level time-frequency (TFR)
+# decomposition for each epoch. We must pass ``output='complex'`` if we wish to
+# use this TFR later with a DICS beamformer. We also pass ``average=False`` to
+# compute the TFR for each individual epoch.
+epochs_tfr = tfr_morlet(epochs, freqs, n_cycles=5, return_itc=False, decim=20,
+                        output='complex', average=False)
+
+# %%
+# Now, we build a DICS beamformer and project the sensor-level TFR to the
+# source level.
+
+# Compute the Cross-Spectral Density (CSD) matrix for the sensor-level TFRs.
+# We are interested in increases in power relative to the baseline period, so
+# we will make a separate CSD for just that period as well.
+csd = csd_tfr(epochs_tfr, tmin=-1, tmax=1.5)
+baseline_csd = csd_tfr(epochs_tfr, tmin=-1, tmax=0)
+
+# Use the CSDs and the forward model to build the DICS beamformer.
+fwd = mne.read_forward_solution(fname_fwd)
+filters = make_dics(epochs.info, fwd, csd, noise_csd=baseline_csd,
+                    pick_ori='max-power', reduce_rank=True, real_filter=True)
+
+# Project the TFR for each epoch to source space
+epochs_stcs = apply_dics_tfr_epochs(epochs_tfr, filters, return_generator=True)
+
+# %%
+# Visualize the first epoch at ~15 Hz.
+stc = next(epochs_stcs)[2]
+
+# At this point, the data is still complex. Convert to power.
+stc.data = np.abs(stc.data)
```

Comment:
I think this is actually amplitude, if I'm not mistaken power would be:

```suggestion
# At this point, the data is still complex. Convert to power.
stc.data = (stc.data * stc.data.conj()).real
```

Reply:
Agreed!